examples/with-react-hook-form/components/Login.js

```diff
@@ -0,0 +1,57 @@
+import { useForm } from 'react-hook-form'
+
+const Login = () => {
+  const { register, errors, handleSubmit } = useForm()
+  const login = handleSubmit(({ username, password, remember }) => {
+    console.log(
+      `username:${username}, password:${password}, remember:${remember}`
```

Comment:
Perhaps do not log the password.

Reply:
yep, i removed console.log completely because comment in the next line is enough i think